src/components/SiteHeader/index.js

```diff
@@ -328,6 +349,20 @@ export const SiteHeader = (props) => {
 
       {tall && <Notification />}
     </>
+  ) : (
+    <IncidentMapHeaderWrapper>
+      <IncidentMapHeader>
+        <Title>
+          <Wrapper>
+            <Logo />
+          </Wrapper>
+          <Wrapper>
+            <h2>Meldingenkaart</h2>
```

Comment:
Is hier geen amsterdam styles components versie van?

Reply:
Nee, de header zoals ik m nu heb gemaakt lijkt eigenlijk meer op de header van de nieuwe stijl. Maar daar zijn de asc nog niet op aangepast.

Reply:
aah okeee! daar ging het over bij gilde right?

Reply:
inderdaad!